src/components/mainBackground/MainBackground.tsx

```diff
@@ -3,14 +3,14 @@ import styled from "styled-components";
 import NetflixBcg from "../../assets/NetflixBcg.jpg";
 
 interface BackgroundProps {
-	imageUrl?: string;
+	imageurl?: string;
 	children?: React.ReactNode;
 }
 
 const MainBackground = styled.div<BackgroundProps>`
 	height: auto;
 
-	background-image: url(${(props) => props.imageUrl});
+	background-image: url(${(props) => props.imageurl});
```

Comment:
Masz dostęp do NetflixBcg na tym poziomie, więc niepotrzebnie przekazujesz go jako props. Będzie miało to wszystko sens, jeżeli przeniesiesz style do osobnego pliku, a tak w sumie powinno się zdarzyć.